client/Fireport/Assets/Scripts/AnimationEngine/AnimationActionFactory.cs

```diff
@@ -0,0 +1,40 @@
+using AnimationEngine.Animations;
+using Game;
+using Model.State;
+
+namespace AnimationEngine {
+    public static class AnimationActionFactory {
```

Comment:
I could have done this as a non-static class that extended ScriptableObject.  That would have been the Unity-UI-Friendly way to do it.  But instead, I went with the hardcoded, but more programmer/git friendly way.  New animations will have to be manually added to this switch statement.